src/Telegram.Bot/Types/VoiceChatScheduled.cs

```diff
@@ -10,10 +10,9 @@ namespace Telegram.Bot.Types
     public class VoiceChatScheduled
     {
         /// <summary>
-        /// Point in time (Unix timestamp) when the voice chat is supposed to be started by a chat administrator
+        /// Point in time when the voice chat is supposed to be started by a chat administrator
         /// </summary>
-        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
```

Comment:
Why did you remove JsonProperty attribute?

Reply:
as I understand it `start_date ` is not optional in [VoiceChatScheduled](https://core.telegram.org/bots/api#voicechatscheduled), but `VoiceChatScheduled` might be optional inside [`Message`](https://core.telegram.org/bots/api#message), so it's marked as optional [there](https://github.com/TelegramBots/Telegram.Bot/blob/925532d241463ade1730e35de4a3220b7545f27b/src/Telegram.Bot/Types/Message.cs#L349-L350)

Reply:
It also lack `[JsonObject(MemberSerialization.OptIn, NamingStrategyType = typeof(SnakeCaseNamingStrategy))]` attribute. We use opt-in strategy everywhere os `JsonProperty` attributes must be applied.